mlflow/models/evaluation/default_evaluator.py

```diff
@@ -600,14 +600,15 @@ def _evaluate_classifier(self):
 
         def plot_confusion_matrix():
             import matplotlib
+            from mlflow.sklearn._confusion_matrix import ConfusionMatrixDisplay
 
             with matplotlib.rc_context(
                 {
-                    "font.size": min(10, 50.0 / self.num_classes),
+                    "font.size": min(10, 80.0 / self.num_classes),
                     "axes.labelsize": 10,
                 }
             ):
-                sk_metrics.ConfusionMatrixDisplay(
+                ConfusionMatrixDisplay(
                     confusion_matrix=confusion_matrix,
                     display_labels=self.label_list,
                 ).plot(cmap="Blues")
```

Comment:
should we: 
1. expose this as a user configurable kwargs statement for generating these plots and logging them
2. if "no" to 1, should we create a coherent custom cmap dict that aligns to the MLflow theme colors so that the plot looks 'unified' with the UX color scheme?